test/module/irohad/consensus/yac/yac_block_storage_test.cpp

```diff
@@ -3,14 +3,14 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include "consensus/yac/storage/yac_block_storage.hpp"
-
 #include <gtest/gtest.h>
-#include <boost/optional.hpp>
+
+#include "consensus/yac/storage/yac_block_storage.hpp"
```

Comment:
Related header file should be first according to google codestyle. Since this file tests `yac_block_storage`, it can be called a related header file. Check other files as well.